src/systemctl/systemctl-logind.c

```diff
@@ -304,24 +304,9 @@ int logind_schedule_shutdown(void) {
         if (r < 0)
                 return r;
 
-        switch (arg_action) {
-        case ACTION_HALT:
-                action = "halt";
-                break;
-        case ACTION_POWEROFF:
-                action = "poweroff";
-                break;
-        case ACTION_KEXEC:
-                action = "kexec";
-                break;
-        case ACTION_EXIT:
-                action = "exit";
-                break;
-        case ACTION_REBOOT:
-        default:
-                action = "reboot";
-                break;
-        }
+        action = action_table[arg_action].verb;
```

Comment:
why not: manager_item_for_handle(arg_action)?

Reply:
It's a different table. When refactoring logind I didn't realize that someting called `action_table` already existed in systemctl. We probably should rename the one in logind to avoid confusion.